### PR TITLE
Added questions in linux-quiz.ts src/data directory

### PR DESCRIPTION
**Pull request by chefgs**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by chefgs** [comment]
This PR is related to linux questions [issue](https://github.com/freeCodeCamp/Developer_Quiz_Site/issues/16)

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -1070,6 +1070,36 @@ const linuxQuiz = [
...
+    Link: "https://en.wikipedia.org/wiki/Basename"
+  },
+  {
+    Question:
+      "In Linux, which command is used to display the amount of available free disk space for file systems?",
+    Answer: "df",
+    Distractor1: "du",
+    Distractor2: "free",
+    Distractor3: "top",
+    Explanation: "df command is used to display the amount of available free disk space for file systems, on which the invoking user has appropriate read access.",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1090]: It is best not to use direct quotes from wikipedia. 

```suggestion
    Explanation: "The df command stands for disk-free and it is used to check the available disk space in Linux.",
```

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -1070,6 +1070,36 @@ const linuxQuiz = [
...
+    Link: "https://en.wikipedia.org/wiki/Df_(Unix)"
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1091]:
```suggestion
    Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/#the-linux-df-command"
```

File: src/data/linux-quiz.ts [hunk 3]
```diff
@@ -1070,6 +1070,36 @@ const linuxQuiz = [
...
   },
+  {
+    Question:
+      "In Linux, which command is used to strip directory and suffix from filenames?",
+    Answer: "basename",
+    Distractor1: "dirname",
+    Distractor2: "pwd",
+    Distractor3: "ls",
+    Explanation: "The basename command can be used to print the filename from the given pathname. It will delete any prefix up to the last forward-slash character and return the result.",
+    Link: "https://en.wikipedia.org/wiki/Basename"
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1081]:
```suggestion
    Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/#the-linux-basename-command"
```

File: src/data/linux-quiz.ts [hunk 4]
```diff
@@ -1070,6 +1070,36 @@ const linuxQuiz = [
...
+  },
+  {
+    Question:
+      "In Linux, which command is used to estimate file space usage?",
+    Answer: "du",
+    Distractor1: "free",
+    Distractor2: "df",
+    Distractor3: "top",
+    Explanation: "du command is used to estimate file space usage. It estimates the space used under a particular directory or files on a file system.",
+    Link: "https://en.wikipedia.org/wiki/Du_(Unix)"
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1101]: we already have a similar command in the quiz.
Please remove this duplicate

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**